libs/datamodel/core/src/configuration/generator.rs

```diff
@@ -1,13 +1,14 @@
 use crate::configuration::preview_features::PreviewFeatures;
+use crate::StringFromEnvVar;
```

Comment:
I think a use of `super::` would be clearer here to show that this type is part of the same module.

Reply:
done